src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceConfiguration.java

```diff
@@ -48,38 +39,6 @@ public void setProjectId(String projectId) {
     this.projectId = projectId;
   }
 
-  public String getParentResourceType() {
-    return parentResourceType;
-  }
-
-  public void setParentResourceType(String parentResourceType) {
-    this.parentResourceType = parentResourceType;
-  }
-
-  public String getParentResourceId() {
-    return parentResourceId;
-  }
-
-  public void setParentResourceId(String parentResourceId) {
-    this.parentResourceId = parentResourceId;
-  }
```

Comment:
Do you know what these parent resource variables were used for?

Reply:
I do not, but they're unused, so I got rid of em.

Reply:
Ok, just checked what we have them set in application.properties - This is where we were setting the location of new google projects. So, makes sense that we no longer need this w/ RBS.